### ISSUE TITLE
"Calculating" would be more descriptive as "Calculating more results"

### ISSUE THREAD

**Issue by sharatisrani** [NCATSTranslator/Feedback#280]
It alternates with "Load New Results" and so that makes for a more logical workflow

"Calculating more results" ---(results show) ----> "Load more results" ---- <click, loads them> ----> "Calculating more results"

till all the ARAs are done

**Comment by sierra-moxon** [comment]
other suggestions today included: "searching..." (with auto reload when new results are present)

**Comment by gglusman** [comment]
On a related topic, when retrieving a previously computed search it might be more appropriate to simply state 'retrieving'.

**Comment by dnsmith124** [comment]
This change has been implemented, waiting on a deploy before I confirm it and close this ticket. 

@gglusman I like this idea once we've managed to speed up the process of retrieving a cached query, right now it still takes a bit longer than a user might think it should. Changing the language to 'retrieving' now might just be salt in that particular wound. 

Definitely will keep it in mind as we improve the speed there 👍
